src/main/java/core/basesyntax/BallColors.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum BallColors {
+    Yellow,
+    Green,
+    Red,
+    Purple,
+    White,
+    Black
```

Comment:
According to the Java style guide, enum values should be in uppercase letters. Please rename the enum constants to follow this convention.